bpf/ut/main_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+// Copyright (c) 2020-2021 Tigera, Inc. All rights reserved.
```

Comment:
Should revert this copyright change, when the file isn't changing in any other way.